src/app/components/list/view-switcher/index.html

```diff
@@ -69,12 +69,12 @@ <h3>Custom views</h3>
     <sky-demo-page-content sectionHeading="Behavior & states">
       <h3>Switch view</h3>
```

Comment:
@Blackbaud-ToddRoberts, should this subheader be “Filters”? This section seems to be about the behavior of filters, so that header seems more precises to me. Or am I misunderstanding what we are calling out here?

Reply:
Updated the header and text. Not sure if the header is exactly right but trying to convey that the data model is preserved when switching.

Reply:
LGTM